style.css

```diff
@@ -1 +1,132 @@
-/*  Add your styling here */
+header {
+    min-height: 30vh;
+    width: 100%;
+    background-repeat: no-repeat;
+  }
+
+ 
+.subtitle {
+    color: black;
+    padding-top: 28vh;
+    padding-left: 50vw;
+
+    
+}
+
+p {
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif
+}
+
+
+  main {
+      padding-top: 5vh;
+      padding-left: 3vh;
+      padding-right: 3vh;
+  }
+
+  footer {
+      background-color: rgb(218, 186, 186);
+      width: 100%;
+      position: absolute;
+      padding-left: 3vh;
+
+  }
+
+  .button-container {
+      display: flex;
+      flex-direction: column;
+      margin-right: 60vw;
+  }
+
+ .btn {
+     width: 25vw;
+ }
+
+
+@media screen and (min-width: 415px) {
+   
+    header {
+        height: 1rem;
+        margin-left: 10rem;
+        margin-right: 10rem;
+    }
+
+    main {
+        margin-left: 10rem;
+        margin-right: 10rem;
+    }
+    
+    nav {
+        margin-top: 3rem !important;
```

Comment:
!important is generally avoided where possible because it can make the CSS behaviour harder to reason about or control but I assume you are fighting against bootstrap or something else you've inherited and cannot control in this case.